tools/system_libs.py

```diff
@@ -264,6 +249,41 @@ def create_wasm_compiler_rt(libname):
     run_commands([[shared.LLVM_AR, 'cr', '-format=gnu', lib] + o_s])
     return lib
 
+  def files_in_path(path_components=None, filenames=None):
+    assert path_components is not None and filenames is not None
```

Comment:
So I can annotate what the function gets called with, otherwise it's less clear at the call site -
 we wind up with something like `foo(['a', 'b', 'c'], ['d', 'e', 'f'])`, and in sufficiently long argument lists it's probably easy to miss the `], [`. Granted half the files end with '.c' and the other half are things like 'system/lib/libc', which is pretty recognizably a path, but I figure this is yet clearer. So, not actually intended to be default arguments at all.
Additionally, I would like python3's `foo(positional_arg, *, keyword_arg=None)` type syntax, which forces the caller to use these as keyword args.